modules/ROOT/pages/design-change-root-file.adoc

```diff
@@ -0,0 +1,80 @@
+
+== Setting the Root File in an API-Specification Project That Contains More Than One Specification
+
+When you import a specification into a project, API Designer asks whether you want to set the specification as the root file of the project.
+
+You can set any specification in an API-specification project as the root file of that project. For example, the following image shows that a project contains two specifications:
+
+image:apid-root-file-api-spec-project-multi-specs.png[An API-specification project with two specifications. The current root file is the RAML version of the specification.]
+
+<1> The OAS version of the Inventory Process API specification.
+<2> The RAML version of the same specification. This version of the specification is currently set as the root file.
+
+When you click the three dots to the right of a specification that is not currently the root file, the menu that appears provides an option for setting that specification as the root file.
+
+image:apid-root-file-api-spec-project-multi-specs-2.png[The Set as root file option at the top of the menu opened by clicking the three dots.]
+
+<1> The three dots that open the action menu for a file when clicked.
+<2> The *Set as root file* option at the top of the action menu.
+
+
+== Root Files in API-Fragment Projects
+
+Every API-fragment project is created with a default root file. You can switch the root file when there is more than one fragment in a project.
+
+== The Default Root File in API-Fragment Projects
+
+When you create an API-fragment project, the file that is created by default is set to be the root file.
+
+image:apid-default-root-file-api-frag-project.png[The default root file in an API-fragment project named "Retail Common".]
+
+<1> The name of the RAML API-fragment project is Retail Common.
+<2> The default file is named `retail-common.raml` and is set as the root file because every API-fragmentation project must have a root file.
+
+== Setting the Root File in an API-Fragment Project That Contains More Than One Fragment
+
+When you import a fragment into an API-fragment project, API Designer asks whether you want to set the fragment as the root file of the project.
+
+You can set any fragment in an API-fragment project as the root file of that project. For example, the following image shows that a project contains two fragments, the first of which is set as the root file:
+
+image:apid-root-file-api-frag-project-multi-frags.png[An API-fragment project with two fragments.]
+
+<1> The fragment that is set as the root file.
+<2> The other fragment.
+
+When you click the three dots to the right of a fragment that is not currently the root file, the menu that appears provides an option for setting that fragment as the root file.
+
+image:apid-root-file-api-frag-project-multi-frags-2.png[The Set as root file option at the top of the menu opened by clicking the three dots.]
```

Comment:
```suggestion
image:apid-root-file-api-frag-project-multi-frags-2.png[The *Set as root file option* at the top of the menu opened by clicking the three dots.]
```

Reply:
Does the * formatting not work inside the image text?
I'll stop marking this. Please evaluate and address GLOBALLY.